packages/expo-face-detector/package.json

```diff
@@ -33,7 +33,7 @@
   "license": "MIT",
   "homepage": "https://docs.expo.io/versions/latest/sdk/facedetector/",
   "jest": {
-    "preset": "expo-module-scripts"
+    "preset": "expo-module-scripts/ios"
```

Comment:
the tests in this package look like they could be made universal pretty easily in another PR